cogs/cmd_interface/admin/score.py

```diff
@@ -19,12 +22,17 @@ def __init__(self, client):
     async def _score_add(self, interaction: discord.Interaction, member: discord.Member, score: int, reason: str):
         await interaction.response.defer(thinking = True)
 
-        lc_col = self.client.DBClient['LC_db']['LC_users']
-        lc_user = lc_col.find_one({'discord_id': member.id})
-        lc_user['current_month']['score'] += score
-        lc_user['all_time']['score'] += score
-        lc_query = {'$set': lc_user}
-        lc_col.update_one({'discord_id': member.id}, lc_query)
+        daily_obj = self.db_api.update_score(str(member.id), score)
+        if (daily_obj == None):
+            await interaction.followup.send(f"The bot is bug-ed in score add, contact bot dev")
```

Comment:
If no `daily_obj` is found, a new daily_obj should be initiated with `score` point.